inst/Description.qml

```diff
@@ -8,7 +8,7 @@ Description
 	description		: qsTr("Apply common Bayesian tests from sufficient statistics")
 	icon			: "analysis-bayesian-ttest.svg"
 	requiresData	: false
-	version			: "0.15"
```

Comment:
can you also change the version in `DESCRIPTION` accordingly?

Reply:
I was too quick approving the PR at first :D

Reply:
Yes definitely + done.